### PR TITLE
Add option to extract single value from BF.INFO

### PR DESCRIPTION
**Pull request by bodevone**
Add option to extract single value from BF.INFO

Related to #389

### CODE REVIEW

File: docs/commands/bf.info.md [hunk 1]
```diff
@@ -4,11 +4,18 @@ Return information about `key` filter.
 
 * **key**: Name of the key to return information about
 
+Optional parameters:
+* **CAPACITY** Capacity
+* **SIZE** Size
+* **FILTERS** Number of filters
+* **ITEMS** Numner of items inserted
+* **ERATE** Expansion rate
```
Comment (ashtul) [inline docs/commands/bf.info.md:12]: Please change to `EXPANSION`.

**Review by ashtul — COMMENTED** [review]
@LiorKogan WDYT?

File: docs/commands/bf.info.md (hunk 1 above)
Comment (bodevone) [reply docs/commands/bf.info.md]: Done

**Review by ashtul — DISMISSED** [review]
LGTM

**Pull request merged**